docs/kafka/nodejs-kafka/README.adoc

```diff
@@ -226,6 +225,46 @@ ifndef::qs[]
 * Verify that the `countries` topic appears on the *Topics* page.
 endif::[]
 
+[id="configuring-access-on-kafka-topic_{context}"]
```

Comment:
@pacostas Thanks so much for adding this detailed procedure on setting access permissions for the `countries` topic.

With that said, I'm not sure this is _quite_ the approach we want to follow here. In general, we continue to add more and more examples and guides for connecting different types of clients to OpenShift Streams. Those examples all use topics with various names and require different access permissions. It's not really scalable for us to write a tailored procedure each time for setting access permissions. It also just increases the number of procedures that we must separately maintain and verify in our documentation set.

We have a [detailed guide](https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/2f4bf7cf-5de2-4254-8274-6bf71673f407#_6b9cf9a6-5bbf-43ab-8ee6-b9f33630bf64) on setting and managing access permissions, and that should be our single point of reference as much as possible in our documentation set. That guide includes example scenarios for setting produce and consume permissions for a specified topic.

Therefore, the changes I recommend here are the following:

- We omit this dedicated procedure on configuring access permissions for the `countries` topic
- In the final `Running the Node.js example application` section, we add one more prerequisite, stating that you must have configured access permissions for the `countries` topic. That should include a link our [access management guide](https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/2f4bf7cf-5de2-4254-8274-6bf71673f407#_6b9cf9a6-5bbf-43ab-8ee6-b9f33630bf64). I will make that suggestion on the relevant line below.

Reply:
Based on the new prerequisite I've suggested above, we should also remove the final prerequisite in the `Configuring the Node.js example application to connect to a Kafka instance` section (line 157). That prereq is not actually needed at that point in the example anyway.

Reply:
I like when everything is included in a tutorial and in case I want to dig further I can follow the references. Mostly I saw it as a friction point, where despite following all the steps on the tutorial, my setup didnt work, and I had to spend time to find some extra pieces.

Although, I totally understand, duplication is not maintainable and should be avoided. I've added your suggestions under this commit https://github.com/redhat-developer/app-services-guides/pull/600/commits/5788cfae2eadac69d06b9adbe7c7d8be9d5af070

@jbyrne-redhat can you approve?